index.html

```diff
@@ -47,10 +47,154 @@ <h1>Hey there. I'm Sodiq. <br> <span>I'm a Software Developer</span></h1>
         </ul>   
       </div>
       
-      <div class="arrow">
+    </section>
+
+    <section class="recent-works"> <!-- flexbox container-->
+      <div class="arrow"> <!-- flexbox item-->
         <i class="fa-solid fa-circle-chevron-down"></i>
       </div>
+
+      <div class="title"> <!-- flexbox item-->
+        <p>My Recent Works</p>
+      </div>
```

Comment:
- [ ] The title 'My Recent Works' should uses a `hx` tag. please, Kindly fix it to the header tag `h1` .